src/breadcrumb/breadcrumb.item.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, HostBinding } from "@angular/core";
+
+/**
+ * A directive for applying styling to an div element.
+ *
+ * Example:
+ *
+ * ```html
+ * <div ibmBreadcrumbItem/>
+ * ```
+ *
+ * See the [vanilla carbon docs](http://www.carbondesignsystem.com/components/text-input/code) for more detail.
+ */
+@Directive({
```

Comment:
@cal-smith the reason I opted for a directive is because the carbon styling didn't work when the DOM contains the component element. 

Using a directive I have DOM like this:
```
<ibm-breadcrumb>
  <nav class="bx--breadcrumb">
    <div ibmbreadcrumbitem="" class="bx--breadcrumb-item">
      <a class="bx--link" href="#"> Breadcrumb 1 </a>
    </div>
  </nav>
</ibm-breadcrumb>
```

Using a component I get this:
```
<nav class="bx--breadcrumb">
  <ibm-breadcrumb-item href="#">
    <div class="bx--breadcrumb-item">
      <a class="bx--link" href="#"></a>
    </div>
  </ibm-breadcrumb-item>
</nav>
```
With the above DOM the Carbon styling doesn't work

Reply:
Interesting, they aren't usually so picky ... In that case would removing the inner div and applying the styles directly to `ibm-breadcrumb-itm` work? You have also have to set the `display` property in the host listener, if `bx--breadcrumb-item` doesn't specify it

Reply:
OK, I have this working now using `ng-template`. Going to investigate supporting the overflow menu before I update the PR.